<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_OpenFile">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>选择目标文件</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_EnvPath1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>D:\Qt\Qt5.9.0\5.9\mingw53_32\bin</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_EnvPath2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>420</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>d:/Qt/Qt5.9.0/Tools/mingw530_32\bin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>设置环境路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_FilePath">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>打包文件路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Pack">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>打包</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Clear">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>清除信息输出</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>10</y>
      <width>323</width>
      <height>28</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>清除命令</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit"/>
    <widget class="QPushButton" name="pushButton_Enter">
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>43</y>
      <width>701</width>
      <height>187</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
